src/Libraries/Microsoft.Extensions.Http.Diagnostics/Logging/HttpClientLoggingTagNames.cs

```diff
@@ -14,47 +14,47 @@ public static class HttpClientLoggingTagNames
     /// <summary>
     /// HTTP Request duration.
     /// </summary>
-    public const string Duration = "duration";
+    public const string Duration = "Duration";
 
     /// <summary>
     /// HTTP Host.
     /// </summary>
-    public const string Host = "httpHost";
+    public const string Host = "server.address";
 
     /// <summary>
     /// HTTP Method.
     /// </summary>
-    public const string Method = "httpMethod";
+    public const string Method = "http.request.method";
 
     /// <summary>
     /// HTTP Path.
     /// </summary>
-    public const string Path = "httpPath";
+    public const string Path = "url.path";
 
     /// <summary>
     /// HTTP Request Body.
     /// </summary>
-    public const string RequestBody = "httpRequestBody";
+    public const string RequestBody = "RequestBody";
 
     /// <summary>
     /// HTTP Response Body.
     /// </summary>
-    public const string ResponseBody = "httpResponseBody";
+    public const string ResponseBody = "ResponseBody";
```

Comment:
even though it's not part of semantic conventions, why aren't we following the same naming rules? ie: `http.response.body`

Reply:
That's what we agreed on:
* if a dimension is a part of the semantic conventions, then we use it;
* otherwise - we use dotnet naming rules for logging dimensions, i.e. `PascalCase`.

Reply:
It looks inconsistent and it'd be great to have the same naming conventions across different attributes. This way user at least won't need to think about casing when writing their queries against logs.

As a side-note, OTel is now incorporating a lot of attributes from ECS - https://www.elastic.co/guide/en/ecs/current/ecs-http.html. There is no promise they will go as is, but they have a lot of relevant attribute such as [http.request.body.bytes](https://www.elastic.co/guide/en/ecs/current/ecs-http.html#field-http-request-body-bytes) or [http.request.body.content](https://www.elastic.co/guide/en/ecs/current/ecs-http.html#field-http-request-body-content)